assets/src/components/RangeSlider.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import 'rc-slider/assets/index.css';
+import 'rc-tooltip/assets/bootstrap.css';
+import {Range} from 'rc-slider';
+
+const rangeSlider = props => {
+    const wrapperStyle = { width: "100%" };
+    const marks = {};
+    
+    for (let week = props.min; week <= props.max; week++) {
+        marks[week] = `${week}`
+    }
+    
+    return (
+        <div style={wrapperStyle}>
+            <p>Select a start and end week</p>
+            <Range 
+            allowCross={false} 
+            min={props.min} 
+            max={props.max}
+            onChange={props.onWeekChange}
+            defaultValue={[props.startWeek, props.endWeek]}
+            marks = {marks} />
+        </div>)
```

Comment:
I feel the slider is spread out the whole page if it is slightly shrunk it would be easy gazing at it.